cmd/upload.go

```diff
@@ -0,0 +1,114 @@
+//
+// Copyright 2021 The Sigstore Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package cmd
+
+import (
+	"fmt"
+
+	"github.com/go-openapi/swag"
+	"github.com/pkg/errors"
+	"github.com/sigstore/helm-sigstore/pkg/chart"
+	"github.com/sigstore/helm-sigstore/pkg/pgp"
+	"github.com/sigstore/helm-sigstore/pkg/rekor"
+	"github.com/sigstore/helm-sigstore/pkg/types"
+	"github.com/sigstore/helm-sigstore/pkg/verifier"
+	"github.com/spf13/cobra"
+)
+
+func NewUploadCmd() *cobra.Command {
+
+	uploadOptions := types.CLIOptions{}
+
+	// uploadCmd represents the upload command
+	uploadCmd := &cobra.Command{
+		Use:   "upload",
+		Short: "Upload Signed Helm Chart",
```

Comment:
Would you like to add the `usage` ? I would expect to see something like `helm sigstore verify <path_to_chart>` like in cosign cli. Instead I see `Usage:  sigstore verify [flags]` which missed the required argument https://github.com/sigstore/helm-sigstore/pull/3/files#diff-6833c134d800a5e41f7be71a3764d426179a965ec90addbbc4fb724ae4890c64R42.

Reply:
Updated